<launch>

<arg name="args" default="$(find tsdf_localization)/../../../mcl_snapshots/mcl_10000__0_5__normal.mcl $(find mcl)/../../../tsdf_maps/avz_map.h5"/>

<node name="num_particles_eval" type="num_particles_eval" pkg="mcl" output="screen" args="$(arg args)">
    <param name="num_particles" type="int" value="10000" />
    <param name="inc" type="int" value="10000" />
    <param name="repeat" type="int" value="1000" />
    <param name="sigma_trans" type="double" value="0.5" />
    <param name="sigma_rot" type="double" value="1.5" />
    <param name="use_cuda" type="bool" value="true" />
    <param name="per_point" type="bool" value="false" />
</node>

</launch>